app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -135,9 +135,21 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
           }
         } else if (slug == Config.flutterSlug) {
           if (const <String>[
+            // Framework test shards that run golden file tests
             'flutter_driver_android_test',
             'framework',
             'misc',
+
+            // Engine test shards that run golden file tests
+            // Monorepo
+            'linux_android_emulator',
+            'linux_host_engine',
+            'linux_web_engine',
+            'mac_host_engine',
+            'mac_unopt',
+
+            // Integration test shards that run golden file tests
```

Comment:
@matanlurey what test shards are using golden files for integration tests? I can add them in here.

Reply:
I only know about `'flutter_driver_android_test'` (listed above).

Reply:
Should we not encode this information into .ci.yaml?

Reply:
Ahh, I see, these are sub-strings that we then check if the name contains...

Reply:
I see this is extending how this worked before, so sgtm

Reply:
I want this to work, but we'd need to make this change carefully (https://github.com/flutter/flutter/issues/143863#issuecomment-2310898513).

Reply:
> I only know about 'flutter_driver_android_test' (listed above).

Fab! So already covered, thanks @matanlurey!

Reply:
> I wanted to use the presence of the [Skia Gold dependency](https://github.com/flutter/flutter/blob/f0bf49559444e3ee66d9d208d4b25ec2d442b8e6/.ci.yaml#L3638) as a signal that the task would report golds

Yeah, I totally agree. I think I recall that the infra team put the gold dependency on a bunch of unrelated checks, so that if they ever needed it, it was there? Or because it was easier to just blanket apply? I don't remember.

Reply:
> > I wanted to use the presence of the [Skia Gold dependency](https://github.com/flutter/flutter/blob/f0bf49559444e3ee66d9d208d4b25ec2d442b8e6/.ci.yaml#L3638) as a signal that the task would report golds
> 
> Yeah, I totally agree. I think I recall that the infra team put the gold dependency on a bunch of unrelated checks, so that if they ever needed it, it was there? Or because it was easier to just blanket apply? I don't remember.

not just them, we all cargo cult copy paste targets :'(